libs/extensions/angular/spot-illustration/src/spot-illustration.component.stories.ts

```diff
@@ -0,0 +1,398 @@
+ *        <circle id="dot-fill" fill="#7BC8DC" cx="24.1428324" cy="23.1144561" r="3.5"></circle>
+ *    </g>
+ * </svg>
+ *
+ * ```
+ *
+ * **Note**: For complete guidelines and naming patterns, request access to the "Spot-illustrations/Guidelines" document in Zeplin.
+ */
+export const ExtendingLibrary: Story = {
+  parameters: {
+    controls: { disable: true },
+    actions: { disable: true },
+  },
+  render: () => ({
+    template: `
+      <div style="padding: 20px;text-align: center;">
+        <h2>📚 Spot Illustration Extension Guide</h2>
+        <p>This documentation story contains all the guidelines needed to extend the spot illustration library.</p>
+        <p><em>All requirements and examples are documented in the story's JSDoc comments above.</em></p>
+      </div>
+    `,
+  }),
```

Comment:
Consider leaving this out of the sidebar with `tags: ['!dev']` on this specific story, instead of creating a custom template.

Then it will only appear on the docs page itself:

```suggestion
  tags: ['!dev']
```

Reply:
That's a nice take on this. Yeah, you guessed right. It was never needed as a story but just in docs. :)